### ISSUE TITLE
rule changes: no-identical-tests.

### ISSUE THREAD

**Issue by Aladdin-ADD** [not-an-aardvark/eslint-plugin-eslint-plugin#17]
check indentical tests despite of property order.

```js
{
  code: 'foo',
  options: ['bar']
},
{
  options: ['bar'],
  code: 'foo'
}
```

expected: no errors.
current: report an error.
